spec/features/book_index_redirect_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Book index redirect', type: :feature do
+  before(:each) do
+    create(:book)
+    Book.reindex
+
+    visit root_path
+  end
+
+  context 'user search:' do
+    it 'specific world' do
+      fill_in 'Book title', with: 'Yard'
+      click_button 'Search'
+      expect(page).to have_current_path('/searches?search=Yard')
+    end
+
+    it 'nothing' do
+      fill_in 'Book title', with: ''
+      click_button 'Search'
+      expect(page).to have_current_path(root_path)
+    end
+  end
+
+  context 'when user pick category:' do
+    it 'all categories' do
```

Comment:
`it 'returns all books' do` -- don't forget about verbs' existence. Also, the main reason for your test is to test BOOKS and you should check if correct books exists in the page